src/containers/pages/MDAPoint/ClientListView/tests/index.test.tsx

```diff
@@ -0,0 +1,163 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
```

Comment:
Can we add a test that the OpenSRP service is called with the right/expected params?

Reply:
https://github.com/onaio/reveal-frontend/issues/889